packages/compass-web/src/entrypoint.tsx

```diff
@@ -284,6 +285,7 @@ const CompassWeb = ({
       enableCreatingNewConnections: false,
       enableGlobalWrites: false,
       optInDataExplorerGenAIFeatures: false,
+      telemetryAnonymousId: new UUID().toString(),
```

Comment:
This would give you a different id every time the component is rendered, even inside a single session, is this really matching what this value is supposed to be? Should we at least persist this to local / session storage?

Reply:
Local storage sounds like a good move. Thanks for the suggestion! Updated.
I think we'll want to pass the `userAuid` from mms to make it consistent. I have that being passed to Compass Web in an upcoming pr I'll tag you in.

Reply:
Updated both `hello` usages to use the `telemetryAtlasUserId` which corresponds to the `userAuid` in mms. Thanks for bringing that up. Let me know how it looks!

Defaulted it to `new UUID().toString()` when not present (it should always be present). We could error there alternatively.